lib/readline.js

```diff
@@ -868,6 +864,14 @@ Interface.prototype._ttyWrite = function(s, key) {
         break;
 
       default:
+        if (key.name === 'tab') {
+          // If tab completion enabled, do that...
+          if (typeof this.completer === 'function') {
+            this._tabComplete();
+            break;
+          }
+        }
```

Comment:
I think if you add a [`// falls through`](http://eslint.org/docs/rules/no-fallthrough.html) comment it will fix that.